### PR TITLE
Netsuite: add change validator to filter out customlist's item removal

### PR DESCRIPTION
**Pull request by omrilit**

### CODE REVIEW

**Comment by omrilit** [comment]
When removing an item from a customlist and deploying it using SDF, the reult of the operation appears as success while nothing really changes in the UI.
There is an open _Enhancement_ on this issue:

> Enhancement 594616: Editing a customlist is misleading

File: packages/netsuite-adapter/src/change_validators/remove_customlist_item.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  ModificationChange,
+} from '@salto-io/adapter-api'
+import { collections } from '@salto-io/lowerdash'
+import { customTypes } from '../types'
+import { CUSTOM_LIST } from '../constants'
+
+const { makeArray } = collections.array
+
+const isCustomListChange = (change: ModificationChange<InstanceElement>): boolean =>
+  getChangeElement(change).type.elemID.isEqual(customTypes[CUSTOM_LIST].elemID)
```
Comment (netama) [inline packages/netsuite-adapter/src/change_validators/remove_customlist_item.ts:27]: nit: should we check the before and after to make sure there wasn't a type change, or is that not possible?

**Review by netama — APPROVED** [review]
LGTM, one small question (might not be an issue)

File: packages/netsuite-adapter/src/change_validators/remove_customlist_item.ts (hunk 1 above)
Comment (omrilit) [reply packages/netsuite-adapter/src/change_validators/remove_customlist_item.ts:27]: Intentionally I check that there was no type change only by elemID. There is another ChangeValidator for filtering out type changes as they are forbidden.

**Pull request merged**